spk/hassio/Makefile

```
@@ -1,10 +1,12 @@
 SPK_NAME = hassio
-SPK_VERS = 20191224
+SPK_VERS = 20200910
 SPK_REV = 1
 SPK_ICON = src/$(SPK_NAME).png
 SPK_DEPENDS = "Docker>=18.09.0-0506"
 DSM_UI_DIR = app
 
+REQUIRED_DSM = 6.2
```

@hgy59 Are you saying that this should be:
```suggestion
ARCH = dockerx64-6.2
```

---

In fact, no. The "perfect way" would be to list all architecture without Docker available in `UNSUPPORTED_ARCHS` but from my point of view `SPK_DEPENDS` is far enough to prevent any mistake/trouble.

---

ARCH=dockerx64
means that DSM is running in a docker container (aka DDSM)